buffer/hybridbuffer/bufferer.go

```diff
@@ -12,7 +12,17 @@ import (
 	"github.com/relex/slog-agent/util"
 )
 
-// bufferer is an intermediate buffer buf which saves log chunks to disk temporarily if needed
+// bufferer is an intermediate buffer buf which saves log chunks to disk temporarily if needed.
+// It consists of two main parts:
+//  1. bufferer.Accept() can be called to push given chunks into the long queue and may unload chunks from memory
```

Comment:
would it also be correct to call the long queue a "persistent queue" and the short queue an in-memory one?